challenges/lib/src/test/java/challenges/utilities/MultiBracketValidationTest.java

```diff
@@ -0,0 +1,29 @@
+package challenges.utilities;
+
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class MultiBracketValidationTest {
+    @Test
+    public void MultiBracketValidationTest(){
```

Comment:
Good work on your tests, testing not only that the basic brackets work. But also throwing more input to ensure it still passes.